examples/nlp/sentence_embeddings_with_sbert.py

```diff
@@ -47,23 +47,26 @@
 Let's install and import the libraries we need. We'll be using the KerasNLP library in
 this example.
 
-We will also enable [mixed perceciosn](https://www.tensorflow.org/guide/mixed_precision)
+We will also enable [mixed precision](https://www.tensorflow.org/guide/mixed_precision)
```

Comment:
I can't comment on correct line, but can you fix this typo: `batchs` -> `batches` (both lowercase and upper case).

Reply:
Oh there is another typo at the bottom:
```
How to work and ean money through internet?
```
`ean` -> `earn`

Reply:
Thanks! Fixed